backend/spindle/utils.py

```diff
@@ -0,0 +1,12 @@
+from aethel.frontend import LexicalPhrase
+
+
+def serialize_phrases_with_infix_notation(
```

Comment:
I needed this for my Sample page as well, so I've extracted it to a separate file. `utils.py` is not the most helpful filename, but this function _is_ a helpful utility function.